docs/dev/release.qmd

```diff
@@ -33,7 +33,7 @@ Don't change the old version numbers in `changelog.qmd`.
 
 ## Update the changelog
 
-The `docs/changelog.qmd` file, hosted on https://deltares.github.io/Ribasim/changelog, records the most important changes for users.
+The `docs/changelog.qmd` file, hosted on https://ribasim.org/changelog, records the most important changes for users.
```

Comment:
We can make it a hyperlink instead of text 
![image](https://github.com/user-attachments/assets/eaa93d8d-938f-4556-87d8-1643b566f43f)